### PR TITLE
Phenocam dataset

### LINKED ISSUE #40: Phenocam

### LINKED ISSUE #71: Got error with daymetr
```bash
Rscript -e 'install.packages("daymetr")'
```

```bash
Error in contrib.url(repos, type) : 
  trying to use CRAN without setting a mirror
Calls: install.packages -> startsWith -> contrib.url
Execution halted
```

### PR DESCRIPTION
**Pull request by sverhoeven**
Fixes #40 
Fixes #71

### CODE REVIEW

File: src/springtime/datasets/phenocam.py [hunk 1]
```diff
@@ -4,72 +4,206 @@
...
+    Install with
+    ```R
+    install.packages("phenocamr")
+    ```
+    """
+
     dataset: Literal["phenocambbox"] = "phenocambbox"
-    area: NamedArea
+    box: Tuple[float, float, float, float]
+    """Bounding box as top left / bottom right pair (maxlat,minlon,minlat,maxlon)."""
```
Comment (khzadeh) [inline src/springtime/datasets/phenocam.py:121]: This way of defining bbox is a bit strange and different from the other datasets. Maybe it is better to be: (minlon, minlat,maxlon,maxlat)

**Review by khzadeh — COMMENTED** [review]
I checked everything in a notebook and seems working fine. It would be nice to have options to get data from multiple points (sites)

File: src/springtime/datasets/phenocam.py (hunk 1 above)
Comment (Peter9192) [reply src/springtime/datasets/phenocam.py:121]: Let's address that in #92

File: src/springtime/datasets/phenocam.py [hunk 2]
```diff
@@ -4,72 +4,206 @@
...
 
 from springtime.config import CONFIG
-from springtime.datasets.daymet import NamedArea
 
 logger = logging.getLogger(__file__)
 
 phenocam_data_dir = CONFIG.data_dir / "phenocam"
 
+
 class PhenocamrSite(BaseModel):
```
[the remark this answers is not in the document]
Comment (Peter9192) [reply src/springtime/datasets/phenocam.py:24]: I think SinglePoint makes most sense for raster data sources. For vector data like phenocam, we could consider adding a nearest site lookup, though I'm not sure if that's really useful. Perhaps the best thing is to document clearly which methods (singlepoint, multipoint, bbox, site) are available for which (raster/vector) data product, and make them consistent across all datasets (see #92)

**Pull request merged**